### DISCUSSION TITLE
Problems with the dpgen run process

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#1069] [Q&A]
Hello，
Four potential functions were trained in the first round of iterations, one of which was not trained well, and whether it was possible to train only three well-trained potential functions in the next iterations.
Thank you!

**Comment by HuangJiameng** [comment]
https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata/numb_models Yes, but 4 is recommend.
